### PR TITLE
Android.mk: Add missing include and flag

### PR DESCRIPTION
**Pull request by vchong**
Signed-off-by: Victor Chong <victor.chong@linaro.org>

### CODE REVIEW

**Comment by vchong** [comment]
Depends on #173.

File: Android.mk [hunk 1]
```diff
@@ -66,13 +66,15 @@ LOCAL_C_INCLUDES += $(LOCAL_PATH)/host/xtest \
...
-		$(LOCAL_PATH)/ta/aes_perf/include
+		$(LOCAL_PATH)/ta/aes_perf/include \
+		$(LOCAL_PATH)/ta/socket/include
 
 # Include configuration file generated by OP-TEE OS (CFG_* macros)
 LOCAL_CFLAGS += -include conf.h
 LOCAL_CFLAGS += -DTA_DIR=\"/system/lib/optee_armtz\"
 LOCAL_CFLAGS += -pthread
 LOCAL_CFLAGS += -g3
+LOCAL_CFLAGS += -D_GNU_SOURCE
```
Comment (jbech-linaro) [inline Android.mk]: Is this really needed with the changes in #173 ? If, yes, then
`Reviewed-by: Joakim Bech <joakim.bech@linaro.org>`

Comment (jbech-linaro) [inline Android.mk]: Or to put it in another way, are you saying that #173 ain't needed if we use the flag here?

Comment (vchong) [inline Android.mk]: No and no. Will remove this.

**Comment by vchong** [comment]
Rebased, commit message updated and tag added. Ok to merge?

Thanks!

**Pull request merged**